### PR TITLE
Add support for sentry

### PR DESCRIPTION
**Pull request by yakky**

### CODE REVIEW

File: roles/app/tasks/main.yml [hunk 1]
```diff
@@ -101,6 +101,17 @@
   tags:
      - deploy
 
+- name: install sentry dependency
+  become_user: "{{ rtd_user }}"
+  pip:
+    name: "{{ item }}"
+    virtualenv: "{{ rtd_virtualenv }}"
+  with_items:
+    - "raven>=6.9<7.0"
```
Comment (xrmx) [inline roles/app/tasks/main.yml]: Not sure we need to pin the version

File: roles/app/templates/project/managed.py [hunk 2]
```diff
@@ -75,6 +77,14 @@ def INSTALLED_APPS(self):  # noqa
         'memory': '999m',
         'time': 3600,
     }
+    {% if SENTRY_DSN|string|length  %}
+
+    import os
```
Comment (xrmx) [inline roles/app/templates/project/managed.py]: not sure we need the os import

File: roles/app/templates/project/managed.py [hunk 3]
```diff
@@ -75,6 +77,14 @@ def INSTALLED_APPS(self):  # noqa
...
+    import raven
+    RAVEN_CONFIG = {
+        'dsn': '{{ SENTRY_DSN }}',
```
Comment (xrmx) [inline roles/app/templates/project/managed.py]: I usually add this so we have the git hash of the running version:
```
        'release': raven.fetch_git_sha(ROOT()),
```

File: roles/app/tasks/main.yml (hunk 1 above)
Comment (yakky) [reply roles/app/tasks/main.yml]: ok

File: roles/app/templates/project/managed.py (hunk 3 above)
Comment (yakky) [reply roles/app/templates/project/managed.py]: Fixed

File: roles/app/templates/project/managed.py (hunk 2 above)
Comment (yakky) [reply roles/app/templates/project/managed.py]: removed

**Pull request merged**